vsts/pipelines/templates/_buildTemplate.yml

```
@@ -43,6 +43,18 @@ steps:
     command: login
     azureSubscriptionEndpoint: ${{ parameters.ascName }}
     azureContainerRegistry: ${{ parameters.acrName }}
+    
+- task: ShellScript@2
```

reordered these to fail fast